src/acceptance-test/java/org/zalando/nakadi/webservice/NakadiControllerAT.java

```diff
@@ -0,0 +1,114 @@
+package org.zalando.nakadi.webservice;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.jayway.restassured.http.ContentType;
+import org.apache.commons.collections.map.HashedMap;
+import org.apache.curator.framework.CuratorFramework;
+import org.apache.http.HttpStatus;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Test;
+import org.zalando.nakadi.config.JsonConfig;
+import org.zalando.nakadi.domain.EventType;
+import org.zalando.nakadi.service.FloodService;
+import org.zalando.nakadi.utils.JsonTestHelper;
+import org.zalando.nakadi.webservice.utils.NakadiTestUtils;
+import org.zalando.nakadi.webservice.utils.ZookeeperTestUtils;
+
+import java.io.IOException;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Set;
+
+import static com.jayway.restassured.RestAssured.given;
+
+public class NakadiControllerAT extends BaseAT {
+
+    private static final String FLOODERS_URL = "/nakadi/flooders";
+    private static final ObjectMapper MAPPER = (new JsonConfig()).jacksonObjectMapper();
+    private static final JsonTestHelper JSON_HELPER = new JsonTestHelper(MAPPER);
+    private static final CuratorFramework CURATOR = ZookeeperTestUtils.createCurator(ZOOKEEPER_URL);
+
+    @After
+    public void setUp() {
+        clearFloodersData();
+    }
+
+    @Test
+    public void testBlockFlooder() throws Exception {
+        final EventType eventType = NakadiTestUtils.createEventType();
+        blockFlooder(new FloodService.Flooder(eventType.getName(), FloodService.Type.CONSUMER_ET));
+
+        Assert.assertNotNull(CURATOR.checkExists()
+                .forPath("/nakadi/flooders/consumers/event_types/" + eventType.getName()));
+    }
+
+    @Test
+    public void testUnblockFlooder() throws Exception {
+        final EventType eventType = NakadiTestUtils.createEventType();
+        final FloodService.Flooder flooder =
+                new FloodService.Flooder(eventType.getName(), FloodService.Type.CONSUMER_ET);
+        blockFlooder(flooder);
+
+        unblockFlooder(flooder);
+
+        Assert.assertNull(CURATOR.checkExists()
+                .forPath("/nakadi/flooders/consumers/event_types/" + eventType.getName()));
+    }
+
+    @Test
+    public void testGetFlooders() throws Exception {
+        final EventType eventType = NakadiTestUtils.createEventType();
+
+        final FloodService.Flooder flooder =
+                new FloodService.Flooder(eventType.getName(), FloodService.Type.CONSUMER_ET);
+        blockFlooder(flooder);
+
+        given()
+                .contentType(ContentType.JSON)
+                .get(FLOODERS_URL)
+                .then()
+                .statusCode(HttpStatus.SC_OK)
+                .content(JSON_HELPER.matchesObject(new HashedMap() {
+                    {
+                        put("consumers", new HashMap<String, Set<String>>() {{
```

Comment:
ahh ok, now I see why you have {{ :)
I think it would be easier to compare the actual json output, not to create these hashmaps.